lib/network/user/user_utils.dart

```diff
@@ -56,6 +56,7 @@ Future _clearStorage(SharedPreferences prefs) async {
   await prefs.setBool(HAS_OPENED, true);
 }
 
+//ignore: prefer-match-file-name
 class UserRepo {
```

Comment:
maybe this can go in its own file

Reply:
For now, I created a user_repository file in the same folder because I think we will change the user repository soon in the authentication flow.